x-pack/plugins/infra/public/components/subscription_splash_content.tsx

```diff
@@ -43,6 +36,12 @@ export const SubscriptionSplashPrompt: React.FC = () => {
   }
 
   const canStartTrial = isTrialAvailable && loadState === 'resolved';
+  const licensePageUrl = http.basePath.prepend(LICENSE_PAGE_PATH);
+  const manageLicenseURL = licenseManagement?.locator
+    ? licenseManagement?.locator?.useUrl({
+        page: 'dashboard',
+      })
+    : licensePageUrl;
```

Comment:
Can you do like this?
```suggestion
  const manageLicenseURL = licenseManagement?.locator?.useUrl({
        page: 'dashboard',
      }) || licensePageUrl;
```